From 464c6dc10f4bb3a7f93ad92fb44a9568e9da6b4f Mon Sep 17 00:00:00 2001
From: Laura Abbott <labbott@redhat.com>
Date: Thu, 14 Dec 2017 17:06:21 -0800
Subject: [PATCH] tools/lib: Remove FSF address
To: Ingo Molnar <mingo@redhat.com>
To: Arnaldo Carvalho de Melo <acme@kernel.org>
Cc: linux-kernel@vger.kernel.org

Checkpatch in the kernel now complains about having the FSF address
in comments. Other tools such as rpmlint are now starting to do the
same thing. Remove the FSF address to reduce warnings on multiple tools.

Signed-off-by: Laura Abbott <labbott@redhat.com>
---
For context, I'm updating the packaging for perf in Fedora which
requires a code/output review. This got flagged during that review.

This part of tools doesn't seem to have a clear owner and I don't think
it's kept in sync with the kernel which is why I split this out.  I don't
particularly care which tree it goes through (perf?)
---
 tools/include/linux/rbtree.h           | 4 ----
 tools/include/linux/rbtree_augmented.h | 4 ----
 tools/lib/rbtree.c                     | 4 ----
 3 files changed, 12 deletions(-)

diff --git a/tools/include/linux/rbtree.h b/tools/include/linux/rbtree.h
index 112582253dd0..b71ec73831bb 100644
--- a/tools/include/linux/rbtree.h
+++ b/tools/include/linux/rbtree.h
@@ -12,10 +12,6 @@
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
 
-  You should have received a copy of the GNU General Public License
-  along with this program; if not, write to the Free Software
-  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-
   linux/include/linux/rbtree.h
 
   To use rbtrees you'll have to implement your own insert and search cores.
diff --git a/tools/include/linux/rbtree_augmented.h b/tools/include/linux/rbtree_augmented.h
index 43be941db695..db08eb26eb15 100644
--- a/tools/include/linux/rbtree_augmented.h
+++ b/tools/include/linux/rbtree_augmented.h
@@ -14,10 +14,6 @@
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
 
-  You should have received a copy of the GNU General Public License
-  along with this program; if not, write to the Free Software
-  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-
   tools/linux/include/linux/rbtree_augmented.h
 
   Copied from:
diff --git a/tools/lib/rbtree.c b/tools/lib/rbtree.c
index 17c2b596f043..df538c4139ce 100644
--- a/tools/lib/rbtree.c
+++ b/tools/lib/rbtree.c
@@ -14,10 +14,6 @@
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
 
-  You should have received a copy of the GNU General Public License
-  along with this program; if not, write to the Free Software
-  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-
   linux/lib/rbtree.c
 */
 
-- 
2.14.3

